plugins/modules/rds_instance.py

```diff
@@ -24,7 +24,7 @@
   # General module options
     state:
         description:
-          - Whether the snapshot should exist or not. I(rebooted) is not idempotent and will leave the DB instance in a running state
+          - I(rebooted) is not idempotent and will leave the DB instance in a running state
             and start it prior to rebooting if it was stopped. I(present) will leave the DB instance in the current running/stopped state,
             (running if creating the DB instance).
```

Comment:
We usually describe what the parameter's supposed to do, so I think we're better updating it to refer to instance rather than snapshots  I suspect the module was originally copied from the RDS snapshot module and we missed cleaning it up.

```suggestion
          - Desired state of the RDS Instance.
          - I(state=rebooted) is not idempotent and will leave the DB instance in a running state
            and start it prior to rebooting if it was stopped. I(state=present) will leave the DB instance in the current running/stopped state,
            (running if creating the DB instance).
```

Additionally `I(...)` is for parameter names (or name=value), if it's just a value it'd be `C(...)`.  This is a bug in the original documentation, but we should fix it while we're at it

Reply:
I'd recommend removing the redundant "Note is not idempotent" from the line below too (GitHub won't let me make a suggestion against line 31)